packages/routing/addon/templates/common.hbs

```diff
@@ -1 +1,6 @@
-{{cardstack-content content=model format="isolated"}}
+{{!-- TODO Let's eventually refactor this "if" condition out --}}
+{{#if isNewContent}}
+  {{cardstack-content content=model format="isolated"}}
+{{else}}
+  {{cardstack-content content=model.primaryCard format="isolated" params=params}}
```

Comment:
I think cards need the ability to modify the query params in the URL too. I mean, they can choose to do a whole transition and go through the routing system and the spaces over again, and _sometimes_ you want that, but other times you just want to change the URL to reflect the new state you're already in (like the user just typed some text into a search field and you want to update `?q=`).

Perhaps in addition to `params=params` here we could have `setParam=setParam` (and the resulting action handler should be smart enough to only let you set your own known query params).

Reply:
sounds good 👍